### ISSUE TITLE
LibreOffice text highlight color wrong

### ISSUE THREAD

**Issue by geckolinux** [themix-project/oomox-gtk-theme#71]
Hi there, I just created a theme with oomox based on Numix, very happy with it except for one problem: In LibreOffice 7.0 beta2 (current version in openSUSE Tumbleweed) the color of highlighted text in Writer and selected cells in Calc is wrong. It is a grey color (#EBEBEB) whereas it should be a pinkish orange color. This problem does not exist with standard Numix. There's also an error when launching `libreoffice` from the terminal, not sure if that has anything to do with it:

`(soffice:4070): Gtk-WARNING **: 14:24:26.863: Theme parsing error: gtk.css:87:51: Invalid name of pseudo-class`

Thanks in advance for the help!

oomox:
![Screenshot at 2020-07-13 14-32-25](https://user-images.githubusercontent.com/2692138/87345599-eaa6b380-c515-11ea-9333-a02c36a04791.png)

Numix:
![Screenshot at 2020-07-13 14-32-51](https://user-images.githubusercontent.com/2692138/87345606-ed090d80-c515-11ea-9f97-bbf3235ffd35.png)

**Comment by geckolinux** [comment]
Thanks @actionless for adding the labels. Do you need any other information from me to make it easier to find the CSS?

I'm attaching the [gtk-3.20/dist/gtk.css](https://github.com/themix-project/oomox-gtk-theme/files/4918878/gtk.css.txt) file if that helps any.

Also I found that I can run the GTK debugger like this:
`env GTK_THEME=oomox-mine GTK_DEBUG=interactive libreoffice`
I couldn't find any useful CSS elements with it, but it might just be because I don't know how to use it.

**Comment by actionless** [comment]
no, thanks, i'm not going to fix this since it's not reproducible in gtk widget factory and so

i just let it open in case if someone would volunteer

**Comment by geckolinux** [comment]
I'm still trying to debug this. This is a fairly major issue, because that same highlight color also is used in LibreOffice for highlighting search results, which need to "pop" and be very visible from the rest of the gray text. But with the gray highlight color it's very hard to see the search results.

So apparently it is pulling the LibreOffice highlight color from the "Textbox Background" color:

![Screenshot at 2020-07-15 07-55-26](https://user-images.githubusercontent.com/2692138/87548932-ce6c5900-c672-11ea-8743-b99e1d2916ed.png)


![Screenshot at 2020-07-15 08-00-10](https://user-images.githubusercontent.com/2692138/87548947-d1ffe000-c672-11ea-96b3-d21306dd9fb1.png)

This issue does not exist when changing it to an Arc based theme, but I don't like the way the Arc widgets look at all.

**Comment by geckolinux** [comment]
> since it's not reproducible in gtk widget factory 

Hi @actionless , I'm sorry, I didn't understand this part? Do you mean that the color is correct in LibreOffice for you?

**Comment by actionless** [comment]
no, i am talking about https://developer.gnome.org/gtk3/stable/gtk3-widget-factory.html
